### PR TITLE
Custom URL for isRailsDirectUpload

### PR DESCRIPTION
**Pull request by mech**
Ability to add custom URL if your Rails API is on another host.

### CODE REVIEW

File: src/components/ReactTrixRTEInput/ReactTrixRTEInput.jsx [hunk 1]
```diff
@@ -23,14 +23,16 @@ function ReactTrixRTEInput(props) {
...
   } = props;
   const trixRTEInputRef = trixInputRef ? trixInputRef : useRef();
   const [value, setValue] = useState(defaultValue);
   const trixRTEInputId = props.id || getUniqInputId();
   const trixRTEInputName = props.name || "content";
   const directUploadOptions = isRailsDirectUpload ? {
-    "data-direct-upload-url": RAILS_DIRECT_UPLOADS_URL,
-    "data-blob-url-template": RAILS_SERVICE_BLOB_URL
+    "data-direct-upload-url": railsDirectUploadUrl || RAILS_DIRECT_UPLOADS_URL,
+    "data-blob-url-template": railsBlobUrlTemplate || RAILS_SERVICE_BLOB_URL
```
Comment (abhaynikam) [inline src/components/ReactTrixRTEInput/ReactTrixRTEInput.jsx:35]:
```suggestion
    "data-blob-url-template": railsBlobUrl || RAILS_SERVICE_BLOB_URL
```

**Comment by abhaynikam** [comment]
@mech 
- We need to add documentation for new props added to ReactTrixRTEInput. You can find documentation in the readme file.
- Changelog entry
- If possible, a storybook example to showcase the usecase for new options. (can be ignored)

**Comment by mech** [comment]
Changed props name and also updated documentation 😀

**Comment by abhaynikam** [comment]
Thanks, @mech. Releasing a new version today/tomorrow EOD.

**Pull request merged**